code/arg_parser.py

```diff
@@ -0,0 +1,30 @@
+import os
+import argparse
+import logging
+        
+def process():
```

Comment:
This can be a Class `ArgParser` with the method name sometime like `processArgs`

Reply:
changed